servus/serializable.h

```diff
@@ -99,23 +100,28 @@ class Serializable
 #endif
 
     /**
-     * Set a new function called after the object has been updated.
-     *
-     * @return the previously set function.
+     * Set a new function called after every change on the local object
+     * instance (e.g. Collage distributable).
      */
-    SERVUS_API ChangeFunc setUpdatedFunction( const ChangeFunc& func );
+    SERVUS_API void setLocalChangeFunction( const ChangeFunc& func );
+
+    /**
+     * Set a new function called after the object has been updated remotely
+     * (via a subscriber, a http server, loading from file...).
+     */
+    SERVUS_API void setRemoteUpdateFunction( const ChangeFunc& func );
```

Comment:
I'm bad with naming, but I thought that specifying the source of the change/update in the name of the method (local or remote) helps to understand the semantics. Maybe not...